misc/python/materialize/cli/mzcompose.py

```diff
@@ -27,6 +27,21 @@
 announce = ui.speaker("==>")
 say = ui.speaker("")
 
+MIN_DOCKER_VERSION = (1, 27, 0)
```

Comment:
nit: `MIN_DOCKER_COMPOSE_VERSION`, since it's versioned separately from Docker.

Reply:
Done!